tasks/interactive-bots.md

```diff
@@ -123,7 +121,7 @@ We recommend completing Task Type A before doing this task.
 Follow the tutorial below to create your first simple bot that sends private
 messages to users.
 
-* Make a copy of `followup.py` under `contrib_bots/lib` and name it `links.py`, to build on existing code.
+* Make a copy of `followup.py` located in `contrib_bots/lib` and name it `links.py`, to build on existing code.
```

Comment:
This the `links bot` part, so `followup.py` is already there in `contrib_bots/lib`